classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php

```diff
@@ -391,7 +399,7 @@ protected function _saveEditorDecision($args, $request, $formName, $redirectOp =
 			$notificationMgr->updateNotification(
 				$request,
 				$notificationTypes,
-				array($submission->getUserId()),
+				$authorUserIds,
```

Comment:
We talked about this during UI/UX and decided that at least for the moment it's OK for everyone with an author role to receive the notifications. Translators are maybe the only confusing situation at the moment -- others (in OMP in particular) are logical: chapter authors, series editors, etc.